docs/intro.asciidoc

```diff
@@ -6,17 +6,23 @@ endif::[]
 [[intro]]
 == Introduction
 
-Welcome to the APM .NET Agent documentation.
+The Elastic APM .NET Agent automatically measures the performance of your application and tracks errors.
+It has built-in support for the most popular frameworks and routers,
+as well as a simple API which allows you to instrument any application.
 
-Elastic APM automatically measures the performance of your application and tracks errors.
-For example, it records spans for database queries and transactions for incoming HTTP requests.
-See <<supported-technologies>> to learn about which frameworks and technologies are supported by Elastic APM .NET agent.
+[float]
+[[how-it-works]]
+=== How does the Agent work?
+
+The agent auto-instruments <<supported-technologies,supported technologies>> and records interesting events,
+like HTTP requests and database queries. To do this, it {CONTENT_HERE}
+This means that for the supported technologies, there are no code changes required.
 
-Spans are grouped in transactions - by default one for each incoming HTTP request.
-But it's possible to create custom transactions not associated with an HTTP request.
+The Agent automatically {CONTENT_HERE}
```

Comment:
```suggestion
The Agent automatically registers callback methods for these built in Diagnostic Source events - or in case of the IIS HTTP module as described on the <<setup,Set up the Agent page>>, the user needs to register the module. With this, the supported frameworks trigger agent code for relevant events to measures their duration and collects metadata (like DB statements), as well as HTTP related information (like the URL, parameters, and headers).
```

Reply:
In the `<<setup,Set up the Agent page>>` link it'd be nice to link to the `setup.html#setup-asp-net` part - not sure how to do that 😿

Reply:
I'll fix the link